src/OAuth2/Controller/TokenController.php

```diff
@@ -51,7 +51,8 @@ public function handleTokenRequest(RequestInterface $request, ResponseInterface
             // server MUST disable caching in headers when tokens are involved
             $response->setStatusCode(200);
             $response->addParameters($token);
-            $response->addHttpHeaders(array('Cache-Control' => 'no-store', 'Pragma' => 'no-cache'));
+            $response->addHttpHeaders(array('Cache-Control' => 'no-store', 'Pragma' => 'no-cache',
+                'Content-Type' => 'application/json'));
```

Comment:
This change should not be needed, since it's set when calling send method on Reponse object.